libchisel/src/remapimports.rs

```diff
@@ -0,0 +1,124 @@
+use std::str;
```

Comment:
Generally this is implicit, not required. Same goes for `Vec` and `String` and other basic stdlib types.